example/lib/example2.dart

```diff
@@ -62,6 +62,8 @@ class _ExampleDemoState extends State<ExampleDemo> {
           Padding(
             padding: const EdgeInsets.all(8.0),
             child: SearchField(
+              suggestionState: SuggestionState.enabled,
+              suggestionAction: SuggestionAction.shift,
               suggestions: _suggestions,
               searchInputAction: TextInputAction.next,
```

Comment:
While using the SearchField I realized this property `searchInputAction`  should be renamed back to textInputAction, I believe it, would be convenient to use if the property and class name is the same.
What do you think?

Reply:
I agree on that.